src/test/java/nextstep/subway/section/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,198 @@
+import static nextstep.subway.line.LineAcceptanceTestFixture.*;
+import static nextstep.subway.section.SectionAcceptanceTestFixture.*;
+import static nextstep.subway.station.StationAcceptanceTest.*;
+import static nextstep.subway.station.StationAcceptanceTestFixture.*;
+import static nextstep.subway.utils.JsonPathUtils.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.AcceptanceTest;
+
+@DisplayName("지하철역 구간 관련 기능")
+public class SectionAcceptanceTest extends AcceptanceTest {
+    private Integer WANGSIPLI_ID;
+    private Integer JUKJUN_ID;
+
+    private Integer LINE_BUNDANG_ID;
+
+    @BeforeEach
+    public void setUp() {
+        super.setUp();
+        WANGSIPLI_ID = ID_추출(지하철역_생성(WANGSIPLI));
+        JUKJUN_ID = ID_추출(지하철역_생성(JUKJUN));
+
+        // when
+        LINE_BUNDANG_ID = ID_추출(지하철_노선_생성(LINE_BUNDANG, WANGSIPLI_ID, JUKJUN_ID));
+    }
+
+    /**
+     * Given 지하철 역과 노선이 등록되어있고
+     * When 지하철역 사이에 새로운 지하철 역을 구간으로 등록하면
+     * Then 지하철 노선 목록 조회 시 생성한 역을 찾을 수 있다
+     */
+    @DisplayName("역 사이에 새로운 역 등록")
+    @Test
+    void addSection() {
+        // given
+        String SEOHYUN = "서현역";
+        Integer SEOHYUN_ID = ID_추출(지하철역_생성(SEOHYUN));
+
+        // when
+        Map<String, String> params = new HashMap<>();
+        params.put("upStationId", WANGSIPLI_ID.toString());
+        params.put("downStationId", SEOHYUN_ID.toString());
+        params.put("distance", "5");
+        지하철_구간_등록(LINE_BUNDANG_ID, params);
+
+        // then
+        ExtractableResponse<Response> response = 지하철_노선_목록_조회();
+
+        assertAll(
+            () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value()),
+            () -> assertThat(노선목록_역목록_이름_추출(response)).contains(WANGSIPLI, SEOHYUN, JUKJUN)
+        );
+    }
+
+    /**
+     * Given 지하철 역과 노선이 등록되어있고
+     * When 지하철 노선에 새로운 지하철 역을 상행 종점으로 구간 등록하면
+     * Then 지하철 노선 목록 조회 시 생성한 역을 찾을 수 있다
+     */
+    @DisplayName("새로운 역을 상행 종점으로 등록")
+    @Test
+    void addSection_upStation() {
+        // given
+        String CHEONGRYANGRI = "청량리역";
+        Integer CHEONGRYANGRI_ID = ID_추출(지하철역_생성(CHEONGRYANGRI));
+
+        // when
+        Map<String, String> params = new HashMap<>();
+        params.put("upStationId", CHEONGRYANGRI_ID.toString());
+        params.put("downStationId", WANGSIPLI_ID.toString());
+        params.put("distance", "5");
```

Comment:
```suggestion
        // when
        Map<String, String> 구간등록_요청_파라미트 = 구간요청파라미터(.. , ..,);
```

검증하고자 하는 테스트 코드에 집중하기 위해 메소드로 분리해볼수 있겠네요!

Reply:
Map 생성 로직도 별개의 메서드로 분리하도록 하겠습니다!